_scala3-reference/other-new-features/explicit-nulls.md

```diff
@@ -537,4 +537,4 @@ Our strategy for binary compatibility with Scala binaries that predate explicit
 and new libraries compiled without `-Yexplicit-nulls` is to leave the types unchanged
 and be compatible but unsound.
 
-[More details](https://dotty.epfl.ch/docs/internals/explicit-nulls.html)
+[More details](https://docs.scala-lang.org/scala3/reference/other-new-features/explicit-nulls.html)
```

Comment:
this link is to the current page - perhaps we can link to the GitHub doc itself as it doesn't appear to be published anywhere: https://github.com/lampepfl/dotty/blob/main/docs/_docs/internals/explicit-nulls.md